valkyrie/lib/valkyrie/storage_adapter.rb

```diff
@@ -3,18 +3,33 @@ module Valkyrie
   class StorageAdapter
     class_attribute :storage_adapters
     self.storage_adapters = {}
+
+    # Add a storage adapter to the registry under the provided short name
+    # @param storage_adapter [Valkyrie::StorageAdapter]
+    # @param short_name [Symbol]
+    # @return [void]
     def self.register(storage_adapter, short_name)
       storage_adapters[short_name] = storage_adapter
     end
 
+    # @param short_name [Symbol]
+    # @return [void]
```

Comment:
I've been leaving `@return` blank for delete, because I haven't standardized what gets returned - mostly because I don't want to implement `#deleted?`